src/TimeStruct.jl

```diff
@@ -9,12 +9,13 @@ import .Base:
 include("structures.jl")
 include("simple.jl")
 include("calendar.jl")
-include("representative.jl")
+include(joinpath("representative", "core_types.jl"))
```

Comment:
I do not think `joinpath` should be used here. It seems unnecessary in this setting, "representative/core_types.jl" is both shorter and easier to understand.

Reply:
Fair enough. We switched to `joinpath()` a while ago. I guess the key reason from our side is to make it platform independent. Then again, we know that the tests are working with or without on both Linux and Windows.